lib/Subscribers/ImportExport/PersonalDataExporters/NewsletterStatsBaseExporter.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace MailPoet\Subscribers\ImportExport\PersonalDataExporters;
+
+use MailPoet\Models\Subscriber;
+
+abstract class NewsletterStatsBaseExporter {
```

Comment:
You don't need to rewrite it now. But we prefer composition over inheritance. So this would be just a regular class that would be used and called in `NewsletterOpensExporter` and `NewsletterClicksExporter`

Reply:
I see. I don't have a strong preference. Could you point me to a place where I can learn more about the reasonings for preferring composition over inheritance?

Reply:
Thanks for the additional details. I will take a look at the links that you mentioned.